flink-python/pyflink/proto/flink-fn-execution.proto

```diff
@@ -73,6 +73,7 @@ message Schema {
     ARRAY = 16;
     MAP = 17;
     MULTISET = 18;
+    TABLE = 19;
```

Comment:
Because the proto doesn't include any info about the function type is TableFunction, We need  extra info to know the coder type or func type in Python.Maybe there would be better solution. What's your suggestion?